src/c2s/mongoose_c2s.erl

```diff
@@ -449,7 +449,7 @@ stream_start_features_after_auth(#c2s_data{host_type = HostType, lserver = LServ
 
 -spec handle_bind_resource(c2s_data(), c2s_state(), exml:element(), jlib:iq()) -> fsm_res().
 handle_bind_resource(StateData, C2SState, El, #iq{sub_el = SubEl} = IQ) ->
-    case jid:resourceprep(exml_query:path(SubEl, [{element, <<"resource">>}, cdata])) of
+    case jid:resourceprep(xml:get_path_s(SubEl, [{elem, <<"resource">>}, cdata])) of
```

Comment:
We are generally trying to move away from the `xml` module, it's one that was coming with ejabberd back in the very old days until we made a proper `exml` package, to which we are trying to fully move. But we haven't really enforced it 😅